src/main/java/bhs/devilbotz/commands/GripperOpen.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package bhs.devilbotz.commands;
+
+import bhs.devilbotz.subsystems.Gripper;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class GripperOpen extends CommandBase {
+  private Gripper _gripper;
```

Comment:
Remove _ in variable:
eg:
gripper instead of _gripper

(update the rest of the references)

Reply:
done